tests/integration-tests/conftest.py

```diff
@@ -204,13 +204,17 @@ def _cluster_factory(cluster_config):
 
 def _write_cluster_config_to_outdir(request, cluster_config):
     out_dir = request.config.getoption("output_dir")
+
+    # sanitize test name
+    test = request.node.nodeid.split("::", 1)
+    test_file = test[0]
+    test_name = test[1].replace("/", "-")
+
     os.makedirs(
-        "{out_dir}/clusters_configs/{test_dir}".format(out_dir=out_dir, test_dir=os.path.dirname(request.node.nodeid)),
+        "{out_dir}/clusters_configs/{test_dir}".format(out_dir=out_dir, test_dir=os.path.dirname(test_file)),
         exist_ok=True,
     )
-    cluster_config_dst = "{out_dir}/clusters_configs/{test_name}.config".format(
-        out_dir=out_dir, test_name=request.node.nodeid.replace("::", "-")
-    )
+    cluster_config_dst = "{out_dir}/clusters_configs/{test_name}.config".format(out_dir=out_dir, test_name=test_name)
```

Comment:
are you sure test_name is enough?

Reply:
Fixed by concatenating test_file and test_name sanitized.